server/sandbox_run_linux.go

```diff
@@ -1005,6 +1020,30 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 	return resp, nil
 }
 
+// linkLogs bind mounts the pods log directory under the specified empty dir volume
+func linkLogs(kubePodUID, emptyDirVolName, namespace, kubeName, mountLabel string) error {
```

Comment:
can we put this in `internal/factory/sandbox` instead? ideally with a unit test?

Reply:
Moved this code to sandbox_link_logs. I see this analogous to sandbox_network.go code. Also, I think adding a bats test will be more useful here.